background.ts

```diff
@@ -287,6 +296,7 @@ Electron.ipcMain.handle('settings-read', () => {
 // This is the synchronous version of the above; we still use
 // ipcRenderer.sendSync in some places, so it's required for now.
 Electron.ipcMain.on('settings-read', (event) => {
+  console.debug(`event settings-read in main: ${ event }`);
```

Comment:
Is this debug logging required?

Reply:
I mean, no logging is required. But given that this epic is about making debugging information available, this is the sort of thing that we would want to add, unless I'm mistaken?

Reply:
I'd say debug-logging a `settings-write` is more useful than a `settings-read`, and we can leave it in the code.